03-programming.Rmd

```diff
@@ -494,100 +446,6 @@ pryr::object_size(ch)
 pryr::object_size(fac)
 ```
 
-## R objects
```

Comment:
Garrett suggested that there wasn't enough room to cover S3 classes and all that I had achieved was an overview. He suggested that the section be deleted. I agree